extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -551,14 +563,13 @@ export class AmpA4A extends AMP.BaseElement {
           if (!creativeParts) {
```

Comment:
It won't let me comment at the correct line but can you please update the promise that returns extractCreativeAndSignature so that its type if @return {!Promise<?AdResponseDef>}